.gitignore

```diff
@@ -68,3 +68,4 @@ nose2-junit.xml
 # ignore local files
 **/settings.py
 .env.*local
+_volumes
```

Comment:
I would suggest to use `.volumes` instead of `_volumes` for hidden folder.

Reply:
changed